smoke_tests.sh

```diff
@@ -1,4 +1,15 @@
 #!/usr/bin/env bash
 set -ex
 
-[ $(curl --write-out %{http_code} --silent --output /dev/null $(hostname)/medium-articles) == 200 ]
+host=$(hostname)
+# host="localhost:8088"
+
+successResponse=$(curl --write-out %{http_code} --silent --output /dev/null ${host}/medium-articles)
+
+if [ "$successResponse" == 200 ]
+then
+    exit 0
+else
+   curl ${host}/medium-articles
+   exit 1
+fi
```

Comment:
@giorgiosironi Before I merge this, can you look over this change.

For some very strange reason after adding this to try and debug `successResponse` equalled 200. I added these lines in to try and get the build to dump out the error message.